### DISCUSSION TITLE
Nuovo utente errore SSL

### DISCUSSION THREAD

**Discussion by nromagna** [indomus/forum#2198] [HUB: Home Assistant]
Ciao, ho creato un nuovo utente in HA (gli ho anche dato permessi di Amm), quindi scaricata l'app ed inserito le credenziali ma appare sempre l'errore:
![image](https://user-images.githubusercontent.com/79841098/209104270-a18cb077-617d-4681-a5ae-201348148d67.png)

Ho altri 2 telefoni, piu pc vari che si collegano senza problemi. É solo su quel teminale/utente li che da problemi. Se mi connetto da pc con quell'utente funziona...
Potrebbe essere termianle troppo vecchio?

Grazie.

**Comment by snakuzzo** [comment]
1. controlla i log di Core quando provi a collegarti da quel device
2. potrebbe essere un problema legato al device "troppo vecchio"
3. non devi per forza usare l'app...prova anche dal browser

**Reply by nromagna** [reply]
Ciao dal browser fuzniona ma escono contiunuamente notifiche:
```
Login attempt failed
Login attempt or request with invalid authentication from 10.0.0.1 (10.0.0.1). See the log for details.

```
Dove 10.0.0.1 è il mio router.

**Reply by snakuzzo** [reply]
L'errore è chiaro, il perché no. Mi spiace. Forse è davvero troppo vecchio il device...non saprei
